devel/ci/githubprb-project.yml

```diff
@@ -0,0 +1,95 @@
+- trigger:
+    name: githubprb
+    triggers:
+      - github-pull-request:
+          admin-list:
+              - <yourusername>
```

Comment:
Apologies for not calling this out too, you'll want to fill out this section. This should include you and perhaps some of the other bodhi maintainers.

You can also add an 'org-list' section, such that pull requests from anyone in the organization (fedora-infra for example) would automatically run tests on their PRs.

Anyone not in the admin-list (folks that can trigger tests, and whose PRs will be automatically tested) or the org-list (folks whose PRs will be automatically tested), will be held until an admin asks for a test in the github comments.